src/index.js

```diff
@@ -1 +1,71 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { createGame, addScore } from './modules/api.js';
+import { getRecentScores, storeScores } from './modules/localstorage.js';
+
+const submitForm = document.querySelector('.add_score');
+const refreshButton = document.querySelector('.refresh button');
+const scoresContainer = document.querySelector('.scores-container');
+const errorMsg = document.querySelector('.error-msg');
+
+let chessGameId;
+let submittedScores = [];
+
+(async function leaderboardApp() {
+  async function refreshScores(scoresArray) {
+    scoresContainer.innerHTML = '';
+
+    scoresArray.sort((a, b) => b.score - a.score);
+
+    scoresArray.forEach((scoreObj) => {
+      const scoreElement = document.createElement('p');
+      scoreElement.textContent = `${scoreObj.user}: ${scoreObj.score}`;
+      scoresContainer.appendChild(scoreElement);
+    });
+  }
+
+  submitForm.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const nameInput = document.querySelector("input[placeholder='Your name']");
+    const scoreInput = document.querySelector("input[placeholder='Your score']");
+
+    const name = nameInput.value.trim();
+    const score = parseInt(scoreInput.value, 10);
+
+    if (!name || !score) {
+      errorMsg.style.display = 'block';
+      return;
+    }
+
+    await addScore(chessGameId, name, score);
+    nameInput.value = '';
+    scoreInput.value = '';
+
+    errorMsg.style.display = 'none';
+    scoresContainer.style.display = 'block';
+
+    submittedScores.push({ user: name, score });
+
+    storeScores(submittedScores);
+    refreshScores(submittedScores);
+  });
+
+  refreshButton.addEventListener('click', () => {
+    errorMsg.style.display = 'none';
+    scoresContainer.style.display = 'block';
+
+    submittedScores = getRecentScores();
+
+    refreshScores(submittedScores);
+  });
```

Comment:
> Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).

- You have done great so far. :+1: :+1: Your scores are being posted to the API, but there are still some changes you need to make before going to the next milestone. :wave: 

- [ ] Currently, you have the `getScores` function implemented but you seem not to be using the function anywhere. According to the project requirement, when a user clicks on the refresh button, you are meant to get all the scores from the API, but it seems you are getting them from the local storage which is not among the requirements. I'll suggest you update this part of your code to follow the requirement. :+1: :+1: